src/NuGet.Core/NuGet.Protocol.Core.v3/HttpHandlerResourceV3Provider.cs

```diff
@@ -79,7 +74,8 @@ private HttpClientHandler TryGetCredentialAndProxy(PackageSource packageSource)
             return new CredentialPromptWebRequestHandler()
```

Comment:
we should put the settings in the CredentialPromptWebRequestHandler default constructor